src/infrastructure/server.ts

```diff
@@ -43,7 +43,7 @@ const initializeServer = async (): Promise<Server> => {
 
 const startServer = async (): Promise<Server> => {
   await server.start();
-  console.log(`Server running at: http://localhost:${server.info.port}`);
+  console.log(`Server running at: http://${server.info.host}:${server.info.port}`);
```

Comment:
This will display as shown in the screenshot. We changed it to write as localhost instead of 0.0.0.0 (This was actually to fix the Docker issue). Would you be okay with it?
<img width="179" alt="image" src="https://github.com/DEFRA/ncea-frontend/assets/131792244/50f57f56-6b5a-418a-bbb3-d11c488e06ca">

Reply:
As we discussed, I have changed in order to enable the GA tracking as it wont work for Localhost.